gabrielgomesml/training_project_client
# Feature/discussion-chat-2

## What I did

- Create a section for users discussions with node websocket;
- Add SASS to the project and work in home responsiviness;

## How to test

- First of all make sure you are running the most updated server version in your machine. You can get it here: https://github.com/gabrielgomesml/training_project_server/pull/3
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Log into your account and you'll be redirect to the homepage;
- Click in the "Discussões" button in the navbar;
- Create a nickname to be used in the chat and after that create a new room;
- In another account (i recommend log into it with an anonymous window) do the same proccess and enter in the room that you create.

- After that, go to Home and check the FilmBox component styled with SASS and some responsiviness in the page.
- Also, check the HOC loader in Home and Admin page (make sure that you have an amount of movies and users added).
